### PR TITLE
 [ENDPOINT] DELETE /members/:id - OT102-92

### PR DESCRIPTION
**Pull request by gabodmc**
[OT102-92](https://alkemy-labs.atlassian.net/browse/OT102-92)

**AS** user
I WANT delete a member
TO leave the organization

**Accepted criteria:** 
DELETE /members/:id - Must validate if it exists and remove it. If it doesn't, throw error.

### Evidence:
**Deleting member**

**Trying to delete a deleted member**

**Member list after delete**

### CODE REVIEW

File: controllers/members.js [hunk 1]
```diff
@@ -21,4 +21,21 @@ module.exports = {
...
+    try {
+      const deletedMember = await deleteMember(req.params.id)
+      endpointResponse({
+        res,
+        message: 'Member succesfully deleted',
+        body: deletedMember,
+      })
+    } catch (error) {
+      next(
+        new ErrorObject(
```
Comment (dvelezroman) [inline controllers/members.js]: cambiar por el errorhandler actual

**Review by dvelezroman — CHANGES_REQUESTED** [review]
usar el nuevo formato de error handler...

File: controllers/members.js (hunk 1 above)
Comment (gabodmc) [reply controllers/members.js]: Hecho

**Review by JonatanGarbuyo — APPROVED** [review]

**Review by dvelezroman — APPROVED** [review]
Excelente

**Review by gabyMangoni — APPROVED** [review]
All Ok

**Pull request merged**